docker-compose.yml

```diff
@@ -29,6 +29,7 @@ services:
       - 3000:3000
     volumes:
       - ./:/usr/src/app
+      - /usr/src/app/node_modules
```

Comment:
What does this line do?

Reply:
It prevents the local `node_modules` folder from being mapped into the container, which causes issues